<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reports</class>
 <widget class="QWidget" name="reports">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="daily_reports">
      <attribute name="title">
       <string>Doanh thu theo ngày</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <widget class="QLabel" name="daily_sum">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="total_bills">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>DOANH THU HÔM NAY:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>TỔNG HÓA ĐƠN:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QChartView" name="view_daily_report"/>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="png_daily">
         <property name="text">
          <string>Xuất file png</string>
         </property>
         <property name="icon">
          <iconset resource="galeries.qrc">
           <normaloff>:/image/png.png</normaloff>:/image/png.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>45</width>
           <height>45</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="retail_sum">
      <attribute name="title">
       <string>Doanh thu tổng hợp</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Lựa chọn:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="8">
        <widget class="QChartView" name="view_sum_report"/>
       </item>
       <item row="3" column="6" colspan="2">
        <widget class="QPushButton" name="png_sum">
         <property name="text">
          <string>Xuất file ảnh</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>png.png</normaloff>png.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>45</width>
           <height>45</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="select_time_for_sum">
         <item>
          <property name="text">
           <string>7 ngày qua</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14 ngày qua</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tháng này</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tháng trước</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3 tháng gần đây</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6 tháng gần đây</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Năm nay</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Năm trước</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Tùy chỉnh:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Từ</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QDateEdit" name="start_date_sum">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Đến</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QDateEdit" name="end_date_sum">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QPushButton" name="search_sum">
         <property name="text">
          <string>Tìm kiếm</string>
         </property>
         <property name="icon">
          <iconset resource="galeries.qrc">
           <normaloff>:/image/search.png</normaloff>:/image/search.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="custom_sum">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="retail_bills">
      <attribute name="title">
       <string>Hóa đơn bán lẻ</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="3" column="2">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Đến</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="6" colspan="4">
        <widget class="QComboBox" name="seller_2">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="10">
        <widget class="QPushButton" name="search_bill">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Tìm kiếm</string>
         </property>
         <property name="icon">
          <iconset resource="galeries.qrc">
           <normaloff>:/image/search.png</normaloff>:/image/search.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="3" colspan="3">
        <widget class="QComboBox" name="select_time_bill">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Hôm nay</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hôm qua</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tuần này</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tuần trước</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tháng này</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tháng trước</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3 tháng gần đây</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6 tháng gần đây</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Năm nay</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Năm trước</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QLabel" name="label_16">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Tìm kiếm nhanh theo tên người bán...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDateEdit" name="start_date_bill">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="seller_1">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QDateEdit" name="end_date_bill">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>NVBH</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="11">
        <widget class="QTableWidget" name="view_bills">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Từ</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Lựa chọn:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>NVBH</string>
         </property>
        </widget>
       </item>
       <item row="5" column="10">
        <widget class="QPushButton" name="excel_bill">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Xuất file excel</string>
         </property>
         <property name="icon">
          <iconset resource="galeries.qrc">
           <normaloff>:/image/excel.png</normaloff>:/image/excel.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="6">
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Tùy chỉnh (có thể xuất file excel kết quả tìm kiếm):</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="employees">
      <attribute name="title">
       <string>Nhân viên</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="3">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="8">
        <widget class="QTableWidget" name="view_employee"/>
       </item>
       <item row="4" column="0" rowspan="3">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Thời gian</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_18">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Tùy chọn:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="6">
        <widget class="QPushButton" name="search_employee">
         <property name="text">
          <string>Tìm kiếm</string>
         </property>
         <property name="icon">
          <iconset resource="galeries.qrc">
           <normaloff>:/image/search.png</normaloff>:/image/search.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="4" colspan="2">
        <widget class="QComboBox" name="select_employee"/>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>NVBH</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QComboBox" name="range_time">
         <item>
          <property name="text">
           <string>Theo ngày</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Theo tháng</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Theo quý</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Theo năm</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="stocks">
      <attribute name="title">
       <string>Tồn kho</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="6" column="1">
        <widget class="QPushButton" name="all_stocks">
         <property name="text">
          <string>Hiện thị toàn bộ sản phẩm</string>
         </property>
         <property name="icon">
          <iconset resource="galeries.qrc">
           <normaloff>:/image/all.png</normaloff>:/image/all.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QComboBox" name="select_stock">
         <item>
          <property name="text">
           <string>Top 5 sản phẩm đang bán chạy</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sản phẩm sắp hết hàng</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sản phẩm tồn kho nhiều</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="0" colspan="5">
        <widget class="QTableWidget" name="view_stocks"/>
       </item>
       <item row="5" column="3">
        <widget class="QPushButton" name="sort">
         <property name="text">
          <string>Tồn kho giảm/tăng dần</string>
         </property>
         <property name="icon">
          <iconset resource="galeries.qrc">
           <normaloff>:/image/increasing_decreasing.png</normaloff>:/image/increasing_decreasing.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Lựa chọn:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3" colspan="2">
        <widget class="QPushButton" name="excel_category">
         <property name="text">
          <string>Xuất file excel</string>
         </property>
         <property name="icon">
          <iconset resource="galeries.qrc">
           <normaloff>:/image/excel.png</normaloff>:/image/excel.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QCharts</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="galeries.qrc"/>
 </resources>
 <connections/>
</ui>
